bom/pom.xml

```diff
@@ -17,17 +17,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.sonatype.oss</groupId>
-    <artifactId>oss-parent</artifactId>
-    <version>7</version>
-    <relativePath />
+    <groupId>io.netty</groupId>
+    <artifactId>netty-parent</artifactId>
```

Comment:
The Google BOM best practices guide (https://jlbp.dev/JLBP-15) suggests not including the parent project if it has a `dependency` section in addition to the `dependencyManagement` section as those dependencies will be brought in to projects which import the BOM. I checked the "effective pom" for Netty parent, `mvn help:effective-pom` and it does include some direct dependencies which would be unexpectedly added to projects which import the BOM.

```
...
        <dependency>
          <groupId>io.projectreactor.tools</groupId>
          <artifactId>blockhound</artifactId>
          <version>1.0.3.RELEASE</version>
        </dependency>
      </dependencies>
    </dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.junit.jupiter</groupId>
        <artifactId>junit-jupiter-api</artifactId>
        <version>5.7.0</version>
        <scope>test</scope>
      </dependency>
...
```

Is there a way to switch back to the sonatype `oss-parent` parent pom and still have the versions resolve?

Reply:
See https://github.com/netty/netty/pull/11682